gcloud-java-resourcemanager/pom.xml

```diff
@@ -27,6 +27,12 @@
       <artifactId>google-api-services-cloudresourcemanager</artifactId>
       <version>v1beta1-rev6-1.19.0</version>
       <scope>compile</scope>
+      <exclusions>
```

Comment:
The cloudresourcemanager artifact uses version 1.19 of google-api-client, which pulls some version of guava-jdk5 (maybe 13, but I'd have to double check).  guava-jdk5 version 13 had an old version of clock, which caused build issues.  The latest api client verison is 1.21, which also pulls guava-jdk5, but it doesn't specify a specific version and I didn't check compatibility with clock.